pydax/_dataset.py

```diff
@@ -182,10 +184,16 @@ def delete(self) -> None:
 
     @property
     def data(self) -> Dict[str, Any]:
-        """Access loaded data objects."""
+        """Access loaded data objects.
+
+        :raises RuntimeError: The dataset files were not downloaded and/or not loaded.
+        :return: Loaded data objects.
```

Comment:
I don't think we should document this way, because from the outside, `data` appears as an attribute, not a function:

https://pydax.readthedocs.io/en/latest/api-references/pydax.dataset.Dataset.html#pydax.dataset.Dataset

Reply:
ok no problem